index.html

```diff
@@ -56,20 +54,6 @@
       </form>
     </main>
 
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
-    <script src="js/index.js"></script>
```

Comment:
Perhaps you should remove this script code, as you already know how to fulfil the project requirements, so you don't need this script anymore.

Reply:
- [ ] Kindly phrase your comments politely, as suggestions instead of orders.
- [ ] Also consider saying why you suggest the changes.

Reply:
I would like to recommend you removing this block of code.
This code links to a script that performs some validations in your html to see if you fulfil the requirements for this project.
Now that you know **HTML & CSS** and follow the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) it would be nice to remove things you don't need anymore.

Reply:
I would like to recommend you removing this block of code.
This code links to a script that performs some validations in your html to see if you fulfil the requirements for this project.
Now that you know HTML & CSS and follow the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) it would be nice to remove things you don't need anymore.